src/app/components/MediaPlayer/utils/buildSettings.ts

```diff
@@ -0,0 +1,58 @@
+import getPlayerProps from '#app/legacy/containers/MediaPlayer/helpers/propsInference';
+import onClient from '#app/lib/utilities/onClient';
+import { BuildConfigProps } from '../types.d';
+
+const isTestURL = () => {
+  let isTestRender = false;
+
+  if (onClient()) {
+    const search = new URLSearchParams(window.location.search);
+    isTestRender =
+      process.env.NODE_ENV !== 'production' &&
```

Comment:
Does this mean we're restricting to only when we're running with `yarn start` only? Might it be easier if we just used presence of renderer_env = test only?